greenplumpython/op.py

```diff
@@ -0,0 +1,92 @@
+"""
```

Comment:
doc style check is not enabled for this new file, see https://github.com/greenplum-db/GreenplumPython/blob/b219b93483176227cdb40966fe4f4464bbf1ac2f/tox.ini#L73

I think we can enable it for all files under `greenplumpython` now.